System/Process/Internals.hs

```diff
@@ -163,6 +165,17 @@ createPipe :: IO (Handle, Handle)
 createPipe = createPipeInternal
 {-# INLINE createPipe #-}
 
+-- ---------------------------------------------------------------------------
+-- createPipeFD
+
+-- | Create a pipe for interprocess communication and return a
+-- @(readEnd, writeEnd)@ `FD` pair.
+--
+-- @since 1.2.1.0
```

Comment:
This should actually be 1.4.2.0. Similarly, can you update the ChangeLog and the .cabal file?